cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -356,6 +362,8 @@ def _int_to_bn(self, num, bn=None):
         be discarded after use.
         """
 
+        assert (bn is None) or (bn is not None and bn != self._ffi.NULL)
```

Comment:
Same question: Is this assert required for this PR or could it be split into a separate one?